File: java/hashing/Hashing1.java
import java.util.*;

public class Hashing1 {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int arrSize = sc.nextInt();
    int[] arr = new int[arrSize];
    for (int i = 0; i < arrSize; i++) {
      arr[i] = sc.nextInt();
    }
    // pre compute
    int[] hash = new int[13];
    for (int num : arr) {
      hash[num] += 1;
    }
    int queryNo = sc.nextInt();
    while (queryNo >0) {
      int query = sc.nextInt();
      // fetch 
      System.out.println(hash[query]);
      queryNo--;
    }
  }
}

File: python/hashing/hashing_1.py
def main():
  arr = []
  arr_size = int(input("Enter your arr size: "))
  count = 0
  while (count < arr_size):
    number = int(input("Enter your element: "))
    arr.append(number)
    count += 1
  
  # pre compute
  hash = [0] * 12
  for num in arr:
    hash[num] += 1
    
  query = int(input("Enter the number of query's you want: "))
  while (query > 0):
    number = int(input("Enter the number you want to check the frequency: "))
    # fetch
    print(hash[number])
    query -= 1


    


if __name__ == "__main__":
  main()